Formula/alure.rb

```diff
@@ -1,10 +1,15 @@
 class Alure < Formula
   desc "Manage common tasks with OpenAL applications"
-  homepage "https://kcat.strangesoft.net/alure.html"
-  url "https://kcat.strangesoft.net/alure-releases/alure-1.2.tar.bz2"
+  homepage "https://kcat.tomasu.net/alure.html"
+  url "https://kcat.tomasu.net/alure-releases/alure-1.2.tar.bz2"
   sha256 "465e6adae68927be3a023903764662d64404e40c4c152d160e3a8838b1d70f71"
   revision 1
 
+  livecheck do
+    url "https://kcat.tomasu.net/alure-releases/"
+    regex(/alure[._-]v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/href=.*?alure[._-]v?(\d+(?:\.\d+)+)(?:[._-]src)?\.t/i)
```

This brings the regex up to current standards. There's also an older tarball that includes `-src` in the name, so this optionally matches that format (in case they return to it in the future).

Reply:
Good callout with `-src`